snaps_openstack/utilities/network_utils.py

```diff
@@ -177,3 +177,50 @@ def mtu(task):
         ret = os.system(ansible_command_restart)
 
     return ret
+def preNic(task):
+    ret = None
+    for host in task.get("HOSTS"):
+        ip = host.get("ip")
+        primary_interface = host.get("primary_interface")
+        target_interface = host.get("target_interface")
+        pre_pb = pkg_resources.resource_filename(
+                'snaps_openstack.utilities.playbooks',
+                'viface_pre_play.yaml')
+        ansible_command = "ansible-playbook " + pre_pb \
+                              + " --extra-vars=\'{\"target\": \"" \
+                              + ip + "\",\"target_interface\": \"" + target_interface + "\"}\'"
+        logger.info(ansible_command)
+        ret = os.system(ansible_command)
+    return ret
+def postNic(task):
+    ret = None
+    for host in task.get("HOSTS"):
+        ip = host.get("ip")
+        primary_interface = host.get("primary_interface")
+        target_interface = host.get("target_interface")
+        node_type= host.get("node_type")
+        post_pb = pkg_resources.resource_filename(
+                'snaps_openstack.utilities.playbooks',
+                'viface_post_play.yaml')
+        ansible_command = "ansible-playbook " + post_pb \
+                              + " --extra-vars=\'{\"target\": \"" \
+                              + ip + "\",\"node_type\":\"" +node_type+"\",\"interface\": \"" + primary_interface + "\"}\'"
```

Comment:
have blank before and after "+" (i.e. "... + ...").  Also this line is too long, make it two lines.